lib/beaker-openstack/version.rb

```diff
@@ -1,3 +1,3 @@
 module BeakerOpenstack
-  VERSION = '0.3.0'
+  VERSION = '0.4.0'
```

Comment:
Please remove this part - that gets changed at release time

Reply:
Done.  Thanks Gene.